dbt/adapters/athena/lakeformation.py

```diff
@@ -24,6 +26,7 @@ class LfTagsConfig(BaseModel):
     enabled: bool = False
     tags: Optional[Dict[str, str]] = None
     tags_columns: Optional[Dict[str, Dict[str, List[str]]]] = None
+    inherited_tags: Optional[List[str]] = []
```

Comment:
[here](https://github.com/dbt-athena/dbt-athena/pull/453/files#diff-2a4222bf2a5dec1666561daee883fda0b338471f92beb09101cff381ce6d54feR39) you transform the list to a set - isn't better to use `Set[str] = {}` directly?

Reply:
@nicor88 I'm not sure it's possible to define set in `dbt_project.yaml` file as yaml does not supports sets AFAIK

Reply:
you are right, yaml doesn't support it, but model config supports it... a bit ambiguous indeed. We are forced to use List I believe...not a biggy.